group_vars/all

```diff
@@ -7,6 +7,9 @@ server_hostname: "{{ domain | default(ansible_default_ipv4.address) }}"
 # Server Timezone
 timezone: Europe/Madrid
 
+# Consul Democracy Timezone
+application_timezone: Madrid
```

Comment:
Since ActiveSupport also accepts TZInfo identifiers (and, internally, it converts something like "Madrid" to something like "Europe/Madrid" in order to pass to TZInfo), what do you think about:

```suggestion
application_timezone: "{{ timezone }}"
```

Then we can probably remove the entry for this variable from the README :thinking:.

Reply:
I didn't know that! Done!